lib/info-extras.js

```diff
@@ -75,59 +72,54 @@ exports.getVideoMedia = body => {
  * @param {string} body
  * @return {Object}
  */
-const userRegexp = /<a href="\/user\/([^"]+)/;
-const verifiedRegexp = /<span .*?(aria-label="Verified")(.*?(?=<\/span>))/;
-exports.getAuthor = body => {
-  let ownerinfo = util.between(body,
-    '<div id="watch7-user-header" class=" spf-link ">',
-    '<div id="watch8-action-buttons" class="watch-action-buttons clearfix">');
-  if (ownerinfo === '') {
-    return {};
+exports.getAuthor = info => {
+  let results = [], videoDetails = {};
+  try {
+    results = info.response.contents.twoColumnWatchNextResults.results.results.contents;
+  } catch (err) {
+    // Do nothing.
+  }
+  let v = results.find(v2 =>
+    v2.videoSecondaryInfoRenderer && v.videoSecondaryInfoRenderer.owner);
```

Comment:
I think there is a typo here, it should be `v2.videoSecondaryInfoRenderer.owner` instead of `v.videoSecondaryInfoRenderer.owner`

Reply:
thanks!